pkgs/top-level/python-packages.nix

```diff
@@ -1221,6 +1221,8 @@ self: super: with self; {
 
   baron = callPackage ../development/python-modules/baron { };
 
+  barcodenumber = callPackage ../development/python-modules/barcodenumber { };
```

Comment:
Please order alphabetically.

Reply:
Addressed in [force-push](https://github.com/NixOS/nixpkgs/compare/f7b3e41bc47396709912797dec5080d2ccc99943..af331302bfa396a9fc0a387522b4aba0f896b535)